tests/sentry/api/endpoints/test_project_filter_details.py

```diff
@@ -45,6 +45,28 @@ def test_put_health_check_filter(self):
         # option was changed by the request
         assert project.get_option("filters:filtered-transaction") == "0"
 
+    def test_put_chunk_load_error_filter(self):
+        """
+        Tests that it accepts to set the ChunkLoadError filter.
+        """
+        org = self.create_organization(name="baz", slug="1", owner=self.user)
+        team = self.create_team(organization=org, name="foo", slug="foo")
+        project = self.create_project(name="Bar", slug="bar", teams=[team])
+
+        project.update_option("filters:chunk-load-error", "0")
+        self.get_success_response(
+            org.slug, project.slug, "chunk-load-error", active=True, status_code=204
+        )
+        # option was changed by the request
+        assert project.get_option("filters:chunk-load-error") == "1"
+
+        project.update_option("filters:chunk-load-error", "1")
```

Comment:
If the option is set to a value, do we need to update it to the same value?

Reply:
It was just to clearly show that we set it to `1` and not leverage the state left by the previous execution but yeah, that could be removed.